ui/src/main/kotlin/org/jetbrains/jewel/ui/painter/ResourcePainterProvider.kt

```diff
@@ -75,24 +76,30 @@ public class ResourcePainterProvider(
         val scope = Scope(density, basePath, classLoaders)
 
         val currentHintsProvider = LocalPainterHintsProvider.current
-        currentHintsProvider.priorityHints(basePath)
+        currentHintsProvider
+            .priorityHints(basePath)
             .forEach { scope.resolveHint(it) }
 
         hints.forEach { scope.resolveHint(it) }
 
-        currentHintsProvider.hints(basePath)
+        currentHintsProvider
+            .hints(basePath)
             .forEach { scope.resolveHint(it) }
 
         val cacheKey = scope.acceptedHints.hashCode() * 31 + LocalDensity.current.hashCode()
 
         if (inDebugMode && cache[cacheKey] != null) {
-            println("Cache hit for $basePath (accepted hints: ${scope.acceptedHints.joinToString()})")
+            JewelLogger
+                .getInstance("JewelStandaloneShowcase")
```

Comment:
```suggestion
                .getInstance(this::java.class)
```

In general I'd save the logger as a field, instead of creating an instance every time.